aeron-archive/src/main/java/io/aeron/archive/ReplaySession.java

```diff
@@ -586,12 +589,36 @@ static boolean notHeaderAligned(
         return isInvalidHeader(buffer, streamId, termId, termOffset);
     }
 
-    private void state(final State newState)
+    private void state(final State newState, final String reason)
     {
-        //System.out.println("ReplaySession: " + epochClock.time() + ": " + state + " -> " + newState);
+        logStateChange(state, newState, sessionId, recordingId, replayPosition, reason);
         state = newState;
     }
 
+    @SuppressWarnings("unused")
+    private void logStateChange(
+        final State oldState,
+        final State newState,
+        final long sessionId,
+        final long recordingId,
+        final long position,
+        final String reason)
+    {
+        //System.out.println("ReplaySession: " + state + " -> " + newState);
+    }
+
+    @SuppressWarnings("unused")
+    private void logStarted(
+        final long sessionId,
+        final long controlSessionId,
+        final long correlationId,
+        final long streamId,
+        final long recordingId,
+        final long startPosition,
+        final String publicationChannel)
+    {
+    }
+
```

Comment:
Not needed. Covered via existing `CMD_IN_REPLAY` and `CMD_IN_START_BOUNDED_REPLAY` events.